audio/pavucontrol/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                pavucontrol
+version             4.0
+license             GPL-2+
+categories          audio
+maintainers         {mit.edu:quentin @quentinmit} \
+                    openmaintainer
+platforms           darwin
+description         PulseAudio Volume Control
+
+long_description    PulseAudio Volume Control is a GTK+ volume control tool for PulseAudio.
+
+homepage            https://freedesktop.org/software/pulseaudio/pavucontrol/
+master_sites        https://www.freedesktop.org/software/pulseaudio/pavucontrol/
+
+use_xz              yes
+
+checksums           rmd160  efb33c4f161e237b8da445fc9d6baff17be251dc \
+                    sha256  8fc45bac9722aefa6f022999cbb76242d143c31b314e2dbb38f034f4069d14e2 \
+                    size    165216
+
+depends_build       port:pkgconfig \
```

Comment:
we've more-or-less standardized on using depends_build-append, depends_lib-append, etc. It doesn't matter in this Portfile, but it's helpful in case someone needs to come along and throw in a PortGroup like legacysupport or cxx11 or some other. Then it becomes important, as these will overwrite the PG settings and we often miss this.